src/commands/command-docs.json

```diff
@@ -18,6 +18,186 @@
+                        },
+                        "doc_flags": {
+                            "description": "an array of documentation flags",
+                            "type": "array",
+                            "items": {
+                                "oneOf": [
+                                    {
+                                        "description": "the command is deprecated.",
+                                        "const": "deprecated"
+                                    },
+                                    {
+                                        "description": "a system command that isn't meant to be called by users.",
+                                        "const": "syscmd"
+                                    }
+                                ]
+                            }
+                        },
+                        "deprecated_since": {
+                            "description": "the Redis version that deprecated the command (or for module commands, the module version)",
+                            "type": "string"
+                        },
+                        "replaced_by": {
+                            "description": "the alternative for a deprecated command.",
+                            "type": "string"
+                        },
+                        "history": {
+                            "description": "an array of historical notes describing changes to the command's behavior or arguments.",
+                            "type": "array",
+                            "uniqueItems": true,
+                            "items": {
+                                "type": "array",
+                                "items": [
+                                    {
+                                        "type": "string",
+                                        "description": "The Redis version that the entry applies to."
+                                    },
+                                    {
+                                        "type": "string",
+                                        "description": "The description of the change."
+                                    }
+                                ]
+                            }
+                        },
+                        "arguments": {
+                            "description": "an array of maps that describe the command's arguments.",
+                            "type": "array",
+                            "items": {
+                                "type": "object",
+                                "properties": {
+                                    "name": {
+                                        "type": "string"
+                                    },
+                                    "type": {
+                                        "type": "string"
+                                    }
+                                },
+                                "additionalProperties": {
```

Comment:
i meant that all of these should be under `properties` (`additionalProperties` is a bool)

Reply:
`additionalProperties` can be a schema, but since all these properties are optional I've mentioned them here. We can change it once we have the support for "required"